rnd/autogpt_server/autogpt_server/data/graph.py

```diff
@@ -85,11 +85,27 @@ def from_db(graph: AgentGraph):
             name=graph.name or "",
             description=graph.description or "",
             nodes=[Node.from_db(node) for node in graph.AgentNodes or []],
-            links=list({
-                Link.from_db(link)
-                for node in graph.AgentNodes or []
-                for link in (node.Input or []) + (node.Output or [])
-            })
+            links=list(
+                {
+                    Link.from_db(link)
+                    for node in graph.AgentNodes or []
+                    for link in (node.Input or []) + (node.Output or [])
+                }
+            ),
+        )
+
+
+class GraphMeta(BaseModel):
```

Comment:
Would it be much simpler to just add is_template to the existing graph model? what's the concern?

Reply:
or make keep this (for simple api response), but the creation is not from the db but from `Graph`. So the model graph is still only 1 and this class is just for request/response.

Reply:
Not not fully decided on weather or not to have another class here, rather than just return empty lists of nodes and links.

What do you think is best?

Reply:
I have made `Graph` a subclass of `GraphMeta`, so this is resolved